tools/importer/import.js

```diff
@@ -269,14 +268,24 @@ export default {
       ], document));
     }
 
-    const cells = [
-      ['Fragment'],
-      ['https://main--macquarie-help-centre--hlxsites.hlx.live/help/fragment/rating-section'],
-    ];
-    const table = WebImporter.DOMUtils.createTable(cells, document);
-    main.append(table);
-
-    table.after(document.createElement('hr'));
+    // const cells = [
+    //   ['Fragment'],
+    //   ['https://main--macquarie-help-centre--hlxsites.hlx.live/help/fragment/rating-section'],
+    // ];
+    // const table = WebImporter.DOMUtils.createTable(cells, document);
+    // main.append(table);
+    // table.after(document.createElement('hr'));
+
+    // feedback section
+    const ratingContent = document.querySelector('.three-column-block .list');
```

Comment:
@catalan-adobe , can we also have a placeholder value here added before the content list, which can be used for autoblocking?

Reply:
I assumed the same, so we are aligned on this! 👍